framework/include/transfers/MultiAppTransfer.h

```diff
@@ -102,7 +102,6 @@ class MultiAppTransfer : public Transfer
    */
   NumericVector<Real> & getTransferVector(unsigned int i_local, std::string var_name);
 
-private:
   // Given local app index, returns global app index.
   std::vector<unsigned int> _local2global_map;
```

Comment:
I think it needs to become protected.  I've used it now in two derived classes (`MultiAppMeshFunctionTransfer` and `MultiAppProjectionTransfer`)